custom_components/versatile_thermostat/translations/sk.json

```diff
@@ -45,6 +45,7 @@
           "heater_entity2_id": "2. spínač ohrievača",
           "heater_entity3_id": "3. spínač ohrievača",
           "heater_entity4_id": "4. spínač ohrievača",
+          "heater_keep_alive": "Switch keep-alive interval in seconds",
```

Comment:
remove. Fallback is done by HA itself. Idem below. If you are sure for Italian, you can keep it.